static/app/utils/replays/replayReader.tsx

```diff
@@ -155,9 +159,11 @@ export default class ReplayReader {
     )
   );
 
-  getNetworkSpans = memoize(() => this.sortedSpans.filter(isNetworkSpan));
+  getNetworkSpans = () => this.networkSpans;
```

Comment:
Should this be sorted?

Reply:
I thought there's no need here, because we're going to sort it inside the network table, and in the timeline it will render either way. I'll double check though in both cases because of the problems that cropped up with the other PR